From 9cdcea20a5079f5ccdacc03ddcf527de35fe964c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=A9rgio=20M=2E=20Basto?= <sergio@serjux.com>
Date: Sun, 30 Nov 2025 01:06:38 +0000
Subject: [PATCH 8/9] Add source and target configuration to
 maven-compiler-plugin.

Allows using javac target version greater than 1.8 instead of the default.
---
 pom.xml     | 4 ++++
 uis/pom.xml | 8 --------
 2 files changed, 4 insertions(+), 8 deletions(-)

diff --git a/pom.xml b/pom.xml
index 10f282da1..b5744c08a 100644
--- a/pom.xml
+++ b/pom.xml
@@ -160,6 +160,10 @@
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-compiler-plugin</artifactId>
 					<version>3.6.2</version>
+					<configuration>
+						<source>${java.version}</source>
+						<target>${java.version}</target>
+					</configuration>
 				</plugin>
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
diff --git a/uis/pom.xml b/uis/pom.xml
index 8caddc557..6e724378e 100644
--- a/uis/pom.xml
+++ b/uis/pom.xml
@@ -33,14 +33,6 @@
             </resource>
         </resources>
         <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <configuration>
-                    <source>${java.version}</source>
-                    <target>${java.version}</target>
-                </configuration>
-            </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
-- 
2.51.1

